data-prepper-plugins/otel-metrics-source/src/test/java/com/amazon/dataprepper/plugins/source/otelmetrics/OTelMetricsSourceTest.java

```diff
@@ -363,6 +364,139 @@ void start_with_Health_configured_includes_HealthCheck_service() throws IOExcept
         verify(serverBuilder, never()).service(eq("/health"),isA(HealthCheckService.class));
     }
 
+    @Test
+    void start_with_Health_configured_includes_HealthCheck_service_Unauthenticated_disabled() throws IOException {
```

Comment:
This test case seems redundant to me as I am not sure what it covers that the following two tests fail to cover.

Reply:
This test confirms that the appropriate decorator was added to the server builder